src/standard/effectBuilder.html

```diff
@@ -104,16 +104,28 @@
       }
     },
 
-    _addComplexObserverEffect: function(observer) {
-      var sig = this._parseMethod(observer);
-      for (var i=0, arg; (i<sig.args.length) && (arg=sig.args[i]); i++) {
-        this._addPropertyEffect(arg.model, 'complexObserver', {
-          method: sig.method,
-          args: sig.args,
-          trigger: arg
-        });
+    _addComplexObserverEffect: (function() {
+      function MalformedObserverException(observer) {
```

Comment:
Just throw a regular `Error` with this message.
There's no need to make a custom error type.